.github/workflows/trigger-e2e-tests.yml

```diff
@@ -0,0 +1,108 @@
+name: Trigger E2E Tests
+
+on:
+  pull_request:
+    types:
+      - ready_for_review
+  workflow_call:
+    inputs:
+      build-tag:
+        required: true
+        type: string
+
+concurrency:
+  # Allow only one workflow per any non-`main` branch.
+  group: ${{ github.workflow }}-${{ github.ref_name }}-${{ github.ref_name == 'main' && github.sha || 'anysha' }}
+  cancel-in-progress: false
+
+env:
+  # A concurrency group that we use for e2e-tests runs, matches `concurrency.group` above with `github.repository` as a prefix
+  E2E_CONCURRENCY_GROUP: ${{ github.repository }}-${{ github.head_ref }}-${{ github.ref_name }}-${{ github.ref_name == 'main' && github.sha || 'anysha' }}
```

Comment:
This should match `E2E_CONCURRENCY_GROUP` from the `build_and_test` workflow

Reply:
fixed